src/main/java/duke/tasks/TaskManager.java

```diff
@@ -0,0 +1,151 @@
+package duke.tasks;
+
+import java.util.ArrayList;
+
+
+public class TaskManager {
+    public static final String BY_DIVIDER = "/by";
+    public static final String AT_DIVIDER = "/at";
+    public static final String LINE_SEPARATOR = "_____________________________";
+    private static int numberOfTasksUndone;
+    private static ArrayList<Task> taskList;
+
+    public TaskManager() {
+        taskList = new ArrayList<>();
+    }
+
+    public static int getNumberOfTasksUndone() {
+        return numberOfTasksUndone;
+    }
+
+    public static int getNumberOfTasksAdded() {
+        return taskList.size();
+    }
+
+    public static ArrayList<Task> getTaskList() {
+        return taskList;
+    }
+
+    /**
+     * Prints the task list
+     *
+     **/
+    public static void printTaskList() {
+        System.out.println(LINE_SEPARATOR);
+        for (int i = 0; i < taskList.size(); i++) {
+            if (taskList.size() == 0) {
+                break;
+            }
+            System.out.print((i + 1) + ". ");
+            System.out.println(taskList.get(i).toString());
+        }
+        System.out.println("Total tasks undone: " + numberOfTasksUndone);
+        System.out.println(LINE_SEPARATOR);
+    }
+
+    /**
+     * Prints the message containing how many tasks are in the list
+     *
+     * @param size the size of the list, how many tasks are in the list
+     */
+    private static void printTaskNumberMessage(int size) {
+        System.out.println("Now you have " + size + " tasks in the list");
+        System.out.println(LINE_SEPARATOR);
+    }
+
+    /**
+     * Deletes a task from the list
+     *
+     * @param args the string the user inputs
+     */
+    public static void deleteTask(String args) {
+        int stringLength = args.length();
+        int taskIndex = Integer.parseInt(args.substring(stringLength - 1));
+
+        System.out.println(LINE_SEPARATOR);
+        System.out.println("Got it! This task was removed:" + System.lineSeparator()
+                + taskList.get(taskIndex - 1).toString());
+
+        // if deleted task was not marked as done, decrease numberOfTasksUndone
+        if (!taskList.get(taskIndex - 1).getStatusIcon().equals("X")) {
+            numberOfTasksUndone--;
+        }
+        taskList.remove(taskIndex - 1);
+
+        printTaskNumberMessage(taskList.size());
+    }
+
+    /**
+     * Store to do tasks in a list.
+     *
+     * @param args  the item after the command the user inputs
+     **/
+    public static void addToDoTaskToList(String args) {
+
+        ToDo t = new ToDo(args);
```

Comment:
`t` is not a descriptive variable name. Avoid single character variable names unless they are used in loops. (consider this comment for other single character variable names in your code as well)